README.md

```diff
@@ -8,3 +8,28 @@ Scripts for the generation and parsing of LOKI BandGem geometry and data:
 `ConvertLOKIRuns.py` takes the path to the folder containing the LOKI/LARMOR runs as `*.toff` files, the path to `coordinate.txt`, the path to the IDF produced in the last section, and the detector_map for transforming data.
```

Comment:
It doesn't look like you specify what the outputs of running this script actually are. How would the user know that this has worked, where would the outputs get put?